bridge/client/app/project-board/project-board.component.spec.ts

```diff
@@ -156,6 +155,24 @@ describe('ProjectBoardComponent', () => {
     });
   });
 
+  it('should not show notification indicator', () => {
+    expect(fixture.nativeElement.querySelector('.notification-indicator.notification-indicator-absolute')).toBeFalsy();
+
+    component.hasUnreadLogs$ = of(true);
+    fixture.detectChanges();
+    expect(fixture.nativeElement.querySelector('.notification-indicator.notification-indicator-absolute')).toBeTruthy();
+
+    component.hasUnreadLogs$ = of(false);
```

Comment:
Split test.
`should not show an notification indicator if hasUnreadLogs is set to false`

Reply:
done